python/ray/autoscaler/commands.py

```diff
@@ -453,7 +453,7 @@ def _exec(updater, cmd, screen, tmux, port_forward=None, with_output=False):
             cmd = " ".join(cmd)
     return updater.cmd_runner.run(
         cmd,
-        allocate_tty=True,
+        interactive=bool(cmd),
```

Comment:
Isn't this always true for non-empty commands?

Reply:
cmd can be `None`, for example if we're only doing port forwarding.

Reply:
Is there any harm in always enabling interactive mode and just removing the flag?

Reply:
You tell me :p (I think you originally introduced the `allocate_tty` flag a few years ago).

There's technically some overhead, but I don't think it's noticeable